python-threatexchange/tests/test_api.py

```diff
@@ -30,3 +32,27 @@ def test_get_threat_privacy_groups_member(self):
         )
 
         self.assertTrue(isinstance(response[0], ThreatPrivacyGroup))
+
+    def test_get_threat_privacy_groups_owner(self):
+        """
+        Assumes that the app (if token is provided) will have at least one
+        privacy group.
+        """
+        response = self.api.get_threat_privacy_groups_owner()
+        self.assertTrue(
+            isinstance(response, collections.abc.Sequence)
+            and not isinstance(response, staticmethod),
+            "API returned something that's not a list!",
+        )
+
+        self.assertTrue(isinstance(response[0], ThreatPrivacyGroup))
+
+    def test_privacy_group_can_use_threat_updates(self):
+        """
+        Test with correct privacy_group_id and wrong privacy_group_id
+        """
+        response = self.api.privacy_group_can_use_threat_updates(
+            PRIVACY_GROUP_ID_CORRECT
+        )
+
+        self.assertTrue(response)
```

Comment:
This feels like more of a canary then a unit test since its asserting with production data which may change. Can we try to find a way to mock the data or else remove?